lib/ex_fizz_buzz_enterprise/router.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ExFizzBuzzEnterprise.Router do
+  use Plug.Router
+  
+  if Mix.env == :dev do
+    use Plug.Debugger
+  end
+
+  plug :match
+  plug :dispatch
+
+  # Root path
+  get "/" do
+    send_resp(conn, 200, "Welcome to FizzBuzz! Would you like to Fizz and/or Buzz today?")
+  end
+  
+  get "/fizzbuzz" do
+    ExFizzBuzzEnterprise.InputWorker.update(%ExFizzBuzzEnterprise.InputState{})
+    ExFizzBuzzEnterprise.InputWorker.execute
+    send_resp(conn, 200, "FizzBuzzing has commenced!")
+  end
+  
+  get "/fizzbuzz/:start_val/:end_val" do
+    ExFizzBuzzEnterprise.InputWorker.update(%ExFizzBuzzEnterprise.InputState{
```

Comment:
Generally you want to hide the GenServer state as an implementation detail.  Maybe instead of `update` we have a function like `reset(start, end)`?